### PR TITLE
Added a first batch of 19 dating services

### PR DESCRIPTION
**Pull request by streitlua**
Is it ok to make a single PR for these 19 services or would you rather have 19 individual PRs?

### CODE REVIEW

**Comment by MattiSG** [comment]
Thanks @streitlua, and thanks for asking about process!

In general, I suggest to have one PR per service, in order to parallelise work and avoid blocking when one needs more work than the others, but in this case where you have a lot already done and being integrated at once, I believe it is fine this way 👍

@clementbiron, it seems we have less pressure than expected on the UFC front, would you have some availability for reviewing this PR? 🙂

**Comment by clementbiron** [comment]
@MattiSG I'll review this PR tomorow 👍

File: declarations/AdopteUnMec.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "AdopteUnMec",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.adopteunmec.com/cgu/",
+      "executeClientScripts": true,
+      "select": ".tab-content"
```
Comment (clementbiron) [inline declarations/AdopteUnMec.json]: You should remove the print CTA, fixed in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/0a33e4e4d50348ad550114c83b8e58f38cbe40a1

File: declarations/Coffee Meets Bagel.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Coffee Meets Bagel",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://coffeemeetsbagel.com/privacy-terms/",
+      "select": "#av_section_2"
```
Comment (clementbiron) [inline declarations/Coffee Meets Bagel.json]: Use `startBefore` and `endBefore` selectors to select the content of the terms of service and privacy policy, fixed in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/8c7a4fe010546473f1f5aad24e48eebfe0d0de5d

**Comment by clementbiron** [comment]
@streitlua Why are some documents in French and others in English?

**Comment by streitlua** [comment]
@clementbiron We tried to use the English version for all documents but in some cases they were not related to the EU / Switzerland jurisdiction (only UK for instance) so we used the French version for those.

**Comment by clementbiron** [comment]
> @clementbiron We tried to use the English version for all documents but in some cases they were not related to the EU / Switzerland jurisdiction (only UK for instance) so we used the French version for those.

ok thanks !

File: declarations/eharmony.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "eharmony",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://www.eharmony.co.uk/termsandconditions/",
+      "select": ".legalTextBox"
```
Comment (clementbiron) [inline declarations/eharmony.json:6]: Unfortunalty i didn't find an easy way to delete the model cancellation form and the end of the terms of service.

Comment (clementbiron) [reply declarations/eharmony.json]: I removed the links to the pdf file, fixed in [﻿﻿9a0a24baba91f739bba346e273ee41335c289314](https://github.com/OpenTermsArchive/services-dating/pull/2/commits/9a0a24baba91f739bba346e273ee41335c289314). I admit this is not the best way to do so, but it's working.

File: declarations/Grindr.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (clementbiron) [inline declarations/Grindr.json:1]: Updated declaration in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/e6a4a193374eb5929083ca5bed70913b91879ffe

File: declarations/LoveScout24.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "LoveScout24",
+  "documents": {
+    "Terms of Service": {
```
Comment (clementbiron) [inline declarations/LoveScout24.json]: Wrong type of document, fixed in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/1e1724137aeb786f52f934767eda3026c9307d1c

File: declarations/Match.json [hunk 6]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Match",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://uk.match.com/pages/misc/cge",
```
Comment (clementbiron) [inline declarations/Match.json]: Wrong fetch url, fixed in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/d976b544c3e87c3626986f094dda0ddde795217e

File: declarations/Mektoube.json [hunk 7]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Mektoube",
```
Comment (clementbiron) [inline declarations/Mektoube.json:2]: Few adjustements in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/447bf293cd943269d9ddebf266069a8034cf6f3f

File: declarations/Zoosk.json [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+  "name": "Zoosk",
+  "importedFrom": "https://github.com/tosdr/tosback2/blob/f762cd4bbb2571272985fed009201a23300ba9b2/rules/zoosk.com.xml",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://docviewer.zoosk.com/legal-tos-en_eu.html",
+      "select": ".terms-of-service-eu"
+    },
+    "Privacy Policy": {
+      "fetch": "https://docviewer.zoosk.com/legal-privacy-en_eu.html",
+      "select": ".legal__inner"
```
Comment (clementbiron) [inline declarations/Zoosk.json]: I removed the cookie settings button, fixed in https://github.com/OpenTermsArchive/services-dating/pull/2/commits/f2f90904dd93f5328530df236809ba14b90f007e

**Review by clementbiron — APPROVED** [review]

**Comment by streitlua** [comment]
@clementbiron Thank you for going through all of these! We'll make sure that the new declarations are more thoroughly tested.

**Comment by clementbiron** [comment]
It's not easy to make service declarations even for the core team, you did it well for a first batch 👍

**Pull request merged**